### PR TITLE
UCT/RDMACM: split the connect/notify callback flag to one per each side

### PR DESCRIPTION
**Pull request by alinask**

### CODE REVIEW

File: src/uct/ib/rdmacm/rdmacm_cm_ep.c [hunk 1]
```diff
@@ -74,7 +75,8 @@ void uct_rdmacm_cm_ep_error_cb(uct_rdmacm_cm_ep_t *cep,
     ucs_assert(status != UCS_OK);
     cep->status = status;
 
-    if (cep->flags & UCT_RDMACM_CM_EP_CONN_CB_INVOKED) {
+    if (cep->flags & (UCT_RDMACM_CM_EP_CLIENT_CONN_CB_INVOKED |
+                      UCT_RDMACM_CM_EP_SERVER_NOTIFY_CB_INVOKED)) {
```
Comment (yosefe) [inline src/uct/ib/rdmacm/rdmacm_cm_ep.c:79]: maybe add helper function uct_rdmacm_ep_is_connected() to unite common code with rdmacm_cm.c 381-382 and rdmacm_cm_ep.c 428-429?

**Comment by alinask** [comment]
'Cannot contact hpc-arm-cavium-jenkins'

bot:pipe:retest

**Comment by alinask** [comment]
bot:pipe:retest

**Comment by yosefe** [comment]
bot:pipe:retest

**Comment by yosefe** [comment]
bot:pipe:retest

**Comment by yosefe** [comment]
bot:pipe:retest

**Comment by alinask** [comment]
bot:pipe:retest

**Comment by alinask** [comment]
@yosefe let's merge?

**Pull request merged**